src/browser/api/browser_view.ts

```diff
@@ -0,0 +1,84 @@
+import { BrowserView, BrowserViewConstructorOptions, Rectangle, AutoResizeOptions, webContents } from 'electron';
+import { Identity } from '../api_protocol/transport_strategy/api_transport_base';
+import { addBrowserView, getBrowserViewByIdentity, getWindowByUuidName, OfView, removeBrowserView } from '../core_state';
+import { getRuntimeProxyWindow } from '../window_groups_runtime_proxy';
+import { BrowserViewOptions, BrowserViewCreationOptions } from '../../../js-adapter/src/api/browserview/browserview';
+import convertOptions = require('../convert_options');
+import {getInfo as getWebContentsInfo} from './webcontents';
+import of_events from '../of_events';
+import route from '../../common/route';
+
+
+const windowCloseListenerMap = new WeakMap();
+
+export interface BrowserViewOpts extends BrowserViewCreationOptions {
+    uuid: string;
+}
+
+export async function create(options: BrowserViewOpts) {
+    if (!options.target) {
+        throw new Error('Must supply target identity');
+    }
+    const targetWin = getWindowByUuidName(options.target.uuid, options.target.name);
+    if (!targetWin) {
+        throw new Error('Target Window could not be found');
+    }
+    const targetOptions = targetWin._options;
+    const fullOptions = Object.assign({}, targetOptions, options);
+    const view = new BrowserView(convertOptions.convertToElectron(fullOptions, false));
+    const ofView = addBrowserView(fullOptions, view);
+    if (!options.target) {
+        throw new Error('BrowserView must have a target');
+    }
```

Comment:
this check is a duplicate now...

Reply:
No. one checks that an identity was provided, the other that the target is a valid window.

Reply:
Sorry, I don't understand. Aren't they checking the same thing, and aren't we not changing the `options.target` object?

Reply:
@tgoc99 is right, I thought it was a duplicate of the targetWin check.

Reply:
Fixed